From b40e34b093ed19ab6eeafaf47e1c22c07bfbc591 Mon Sep 17 00:00:00 2001
From: Paul Barrette <paul.barrette@freescale.com>
Date: Tue, 1 Dec 2009 15:34:02 -0500
Subject: [PATCH 1/1] Changes for nov 2009
 The build now uses the CROSS_COMPILE and ARCH envs.
 modprobe.conf now contains tle and ipt_l87pm options.
 Currently, there is no way to support mulitple
 /etc/modprobe.conf files in ltib.  fsl_pme_8572, ipt_l7pm and fsl_tlu
 all require additions to modprobe.conf.  Until this is fixed, the
 fsl_pme modprobe.conf will be used as the archtype.


Signed-off-by: Paul Barrette <paul.barrette@freescale.com>
---
 fsl_pme/Makefile                |  118 ++------------------------------------
 fsl_pme/README                  |   14 +++-
 fsl_pme/common.h                |    2 +-
 fsl_pme/include/cli.h           |    2 +-
 fsl_pme/include/common-user.h   |    2 +-
 fsl_pme/include/db.h            |    2 +-
 fsl_pme/include/dll.h           |    2 +-
 fsl_pme/include/driver.h        |    2 +-
 fsl_pme/include/fsl_pme_local.h |    2 +-
 fsl_pme/include/generic_types.h |    2 +-
 fsl_pme/include/hndl.h          |    2 +-
 fsl_pme/include/idx.h           |    2 +-
 fsl_pme/include/log.h           |    2 +-
 fsl_pme/include/mem.h           |    2 +-
 fsl_pme/include/pm_defs.h       |    2 +-
 fsl_pme/include/pm_defs_priv.h  |    2 +-
 fsl_pme/include/pmapp.h         |    2 +-
 fsl_pme/include/pmci.h          |    2 +-
 fsl_pme/include/pmla.h          |    2 +-
 fsl_pme/include/pmll.h          |    2 +-
 fsl_pme/include/pmp.h           |    2 +-
 fsl_pme/include/pmrec.h         |    2 +-
 fsl_pme/include/pmsm.h          |    2 +-
 fsl_pme/include/pmsrc.h         |    2 +-
 fsl_pme/include/pmu.h           |    2 +-
 fsl_pme/misc.agent              |    2 +-
 fsl_pme/modprobe.conf           |    8 ++-
 fsl_pme/pm_scan_demo.c          |    2 +-
 fsl_pme/pmci.c                  |    2 +-
 fsl_pme/pme_dev_init            |    2 +-
 fsl_pme/pmla_common.c           |    2 +-
 fsl_pme/pmla_local.c            |    2 +-
 fsl_pme/pmm.c                   |    2 +-
 fsl_pme/pmsm.c                  |    2 +-
 fsl_pme/util.c                  |    2 +-
 fsl_pme/util.h                  |    2 +-
 fsl_pme/util_kill.c             |    2 +-
 fsl_pme/util_load.c             |    2 +-
 fsl_pme/util_monitor.c          |    2 +-
 fsl_pme/util_reg.c              |    2 +-
 fsl_pme/util_reset.c            |    2 +-
 fsl_pme/util_unload.c           |    2 +-
 fsl_pme/util_writeback.c        |    2 +-
 43 files changed, 64 insertions(+), 156 deletions(-)

diff --git a/fsl_pme/Makefile b/fsl_pme/Makefile
index 0d65dd8..f352926 100644
--- a/fsl_pme/Makefile
+++ b/fsl_pme/Makefile
@@ -1,6 +1,6 @@
 # ***********************< BEGIN COPYRIGHT >************************
 # 
-#  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+#  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
 # 
 #   NOTICE: The information contained in this file is proprietary 
 #   to Freescale Semiconductor and is being made available to 
@@ -9,144 +9,40 @@
 #   under the terms of the license agreement. 
 # 
 # ***********************< END COPYRIGHT >**************************
-#
-# -----=[BEGIN CONFIGURABLE MACROS]=-----
-# Cross compiler prefix.  This compiler must be on the $PATH variable.
-#
-CROSS_COMPILE_PREFIX := powerpc-linux-gnu-
-
-#
-# This is the path to the directory containing a lib and include directory.
-# These two directories contain the cross compiled system libraries like
-# libedit and libncurses need by the sample applications.
-#
-SYS_PATH := /proj/tools/dev-tools/ppc_tool_chain/gcc-4.2.82-eglibc-2.5.82-2
-#
-# -----=[END CONFIGURABLE MACROS]=-----
-#
-
-#
-# -----=[MACROS]=-----
-#
-CC := $(CROSS_COMPILE_PREFIX)gcc
-AR := $(CROSS_COMPILE_PREFIX)ar
-
-# Library and include paths in user and the global include dir
+CC       := $(CROSS_COMPILE)gcc
+AR       := $(CROSS_COMPILE)ar
 LIB_PATH := $(PWD)/lib 
 INC_PATH := $(PWD)/include $(dir $(PWD))/include
-
-# build flags
 CFLAGS   := -O3 -Winline -W -Wall -Wmissing-prototypes -Wshadow 
 CFLAGS   += -Wstrict-prototypes
-
-# compile time definitions
 DEFINES  := -DPLATFORM_8572_ds_git
-
-# System level libraries and include paths
 LIB_PATH += $(SYS_PATH)/lib
 INC_PATH += $(SYS_PATH)/include
-#
-# -----=[END OF MACROS]=-----
-#
 
-# -----=[ HOW TO BUILD AN APPLICATION ]=------
-# 
-# 1. Append the application target name to the  APPS macro:
-#     APPS +=  MYAPP
-#
-# 2. Define a macro based on the target name with the suffix _objs and assign to
-# it all the objects to build.  Each object corresponds to a C file, just the
-# suffix changes to .o, e.g.:
-#    MYAPP_objs := <list of objects, e.g. main.o>
-#
-# 3. Define a macro based on the target name with the suffix _link_libs.  Each
-# value must NOT contain the "lib" prefix or the .so|.a suffix, just the lib
-# short name, e.g.:
-#    MYAPP_link_libs := MYLIB edit pthreads 
-#
-# Repeat steps 1-3 above for a new application, which must have a different
-# name.  The APPS array will be walked by the foreach loop below and generate a
-# dynmic rule for each member of the array.
-#
-#
-# -----=[ HOW TO BUILD A LIBRARY ]=-----
-#
-# 1. Append a lib short name i.e. pthreads, not libpthreads.a, to the LIBS
-# macro:
-#     LIBS += MYLIB
-# 2. Define a macro based on the lib name with the suffix _objs and assign to it
-# all the objects to build:
-#     MYLIB_objs := <list of objects, e.g. test.o test2.o>
-
-#
-# -----=[SAMPLE APP 1: pm_local]=-----
-# 
 APPS             := pmm
 pmm_objs         := pmm.o
-
-
-# link libs from the release
 pmm_link_libs    := swutilities pmll regex pmlalocal pmci statefulrules
-
-# link system libs
 pmm_link_libs    += edit ncurses pthread rt
 
-#
-# -----=[SAMPLE APP 2: pm_scan_demo]=-----
 APPS                   += pm_scan_demo
 pm_scan_demo_objs      := pm_scan_demo.o
-
-# link libs from the release
 pm_scan_demo_link_libs := swutilities pmci
-
-# link system libs
 pm_scan_demo_link_libs +=
 
-#
-# -----=[SAMPLE APP 3: pm_util]=-----
 APPS              += pm_util
 pm_util_objs      := util.o util_reg.o util_kill.o util_reset.o util_load.o
 pm_util_objs      += util_unload.o util_monitor.o util_writeback.o
 
-#
-# -----=[SAMPLE APP 4: pmsmd]=-----
 APPS              += pmsmd
 pmsmd_objs        := pmsm.o
-
-# link libs from the release
 pmsmd_link_libs   := swutilities
-
-# link system libs
 pmsmd_link_libs   += pthread
 
-# requirs no libs 
-
-
-
-# -----[ END OF SAMPLE APPS]-----
-#
-
-#
-# -----=[SAMPLE LIB 1: ]=-----
-#
-LIBS                   := pmci
-pmci_objs              := pmci.o
-
-#
-# -----=[SAMPLE LIB 2: ]=-----
-#
-LIBS                   := pmlalocal
-pmlalocal_objs         := pmla_common.o pmla_local.o
+LIBS              := pmci
+pmci_objs         := pmci.o
 
-
-# -----=[END OF SAMPLE LIBS ]=-----
-#
-
-#
-# -----[ ADD NEW APPLICATIONS HERE ]-----
-
-# -----[ END OF APPLICATIONS ]-----
-#
+LIBS              := pmlalocal
+pmlalocal_objs    := pmla_common.o pmla_local.o
 
 # -------------------------------------------------------
 # -----[ NOTHING BELOW THIS LINE SHOULD BE TOUCHED ]-----
diff --git a/fsl_pme/README b/fsl_pme/README
index ed88fd1..8820eaf 100644
--- a/fsl_pme/README
+++ b/fsl_pme/README
@@ -1,6 +1,6 @@
            README for PME (Pattern Matcher Engine) application
 
-Copyright (C) 2008 Freescale Semiconductor, Inc. All rights reserved.
+Copyright (C) 2009 Freescale Semiconductor, Inc. All rights reserved.
 
 This directory contains the application for Pattern Matcher Engine. 
 
diff --git a/fsl_pme/common.h b/fsl_pme/common.h
index 794b503..f03a871 100644
--- a/fsl_pme/common.h
+++ b/fsl_pme/common.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/cli.h b/fsl_pme/include/cli.h
index 4c41ce2..3f25bbb 100644
--- a/fsl_pme/include/cli.h
+++ b/fsl_pme/include/cli.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/common-user.h b/fsl_pme/include/common-user.h
index d3e6f65..40b76de 100644
--- a/fsl_pme/include/common-user.h
+++ b/fsl_pme/include/common-user.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/db.h b/fsl_pme/include/db.h
index f35f1c1..4e447da 100644
--- a/fsl_pme/include/db.h
+++ b/fsl_pme/include/db.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/dll.h b/fsl_pme/include/dll.h
index 315e0f6..4440286 100644
--- a/fsl_pme/include/dll.h
+++ b/fsl_pme/include/dll.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/driver.h b/fsl_pme/include/driver.h
index 6a8a9f3..85892b9 100644
--- a/fsl_pme/include/driver.h
+++ b/fsl_pme/include/driver.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/fsl_pme_local.h b/fsl_pme/include/fsl_pme_local.h
index b9ae57d..970bd10 100644
--- a/fsl_pme/include/fsl_pme_local.h
+++ b/fsl_pme/include/fsl_pme_local.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/generic_types.h b/fsl_pme/include/generic_types.h
index 6740095..17f63ed 100644
--- a/fsl_pme/include/generic_types.h
+++ b/fsl_pme/include/generic_types.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/hndl.h b/fsl_pme/include/hndl.h
index 5fe37f6..127e01b 100644
--- a/fsl_pme/include/hndl.h
+++ b/fsl_pme/include/hndl.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/idx.h b/fsl_pme/include/idx.h
index 51ff75f..23146a2 100644
--- a/fsl_pme/include/idx.h
+++ b/fsl_pme/include/idx.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/log.h b/fsl_pme/include/log.h
index 1064869..be11d86 100644
--- a/fsl_pme/include/log.h
+++ b/fsl_pme/include/log.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/mem.h b/fsl_pme/include/mem.h
index e390fca..57cacbc 100644
--- a/fsl_pme/include/mem.h
+++ b/fsl_pme/include/mem.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pm_defs.h b/fsl_pme/include/pm_defs.h
index afab21f..56bf0cf 100644
--- a/fsl_pme/include/pm_defs.h
+++ b/fsl_pme/include/pm_defs.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pm_defs_priv.h b/fsl_pme/include/pm_defs_priv.h
index a2a95e9..11e6cd0 100644
--- a/fsl_pme/include/pm_defs_priv.h
+++ b/fsl_pme/include/pm_defs_priv.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pmapp.h b/fsl_pme/include/pmapp.h
index 4e2e9a1..10d623e 100644
--- a/fsl_pme/include/pmapp.h
+++ b/fsl_pme/include/pmapp.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pmci.h b/fsl_pme/include/pmci.h
index 0de7772..a583ecd 100644
--- a/fsl_pme/include/pmci.h
+++ b/fsl_pme/include/pmci.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pmla.h b/fsl_pme/include/pmla.h
index 19bdd1a..7ed89a7 100644
--- a/fsl_pme/include/pmla.h
+++ b/fsl_pme/include/pmla.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pmll.h b/fsl_pme/include/pmll.h
index 1fa8970..1cecb67 100644
--- a/fsl_pme/include/pmll.h
+++ b/fsl_pme/include/pmll.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pmp.h b/fsl_pme/include/pmp.h
index 483013c..4c385c5 100644
--- a/fsl_pme/include/pmp.h
+++ b/fsl_pme/include/pmp.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pmrec.h b/fsl_pme/include/pmrec.h
index fd93684..fcfb593 100644
--- a/fsl_pme/include/pmrec.h
+++ b/fsl_pme/include/pmrec.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pmsm.h b/fsl_pme/include/pmsm.h
index 191e4b8..b8b8309 100644
--- a/fsl_pme/include/pmsm.h
+++ b/fsl_pme/include/pmsm.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pmsrc.h b/fsl_pme/include/pmsrc.h
index f370db1..7425335 100644
--- a/fsl_pme/include/pmsrc.h
+++ b/fsl_pme/include/pmsrc.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/include/pmu.h b/fsl_pme/include/pmu.h
index fd80cf9..56c7447 100644
--- a/fsl_pme/include/pmu.h
+++ b/fsl_pme/include/pmu.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/misc.agent b/fsl_pme/misc.agent
index 0dc9767..e058b87 100755
--- a/fsl_pme/misc.agent
+++ b/fsl_pme/misc.agent
@@ -1,7 +1,7 @@
 #!/bin/bash
 # ************************< BEGIN COPYRIGHT >************************
 #
-# Copyright (C) 2008 Freescale Semiconductor, Inc. All rights reserved.
+# Copyright (C) 2009 Freescale Semiconductor, Inc. All rights reserved.
 #
 #   NOTICE: The information contained in this file is proprietary 
 #   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/modprobe.conf b/fsl_pme/modprobe.conf
index bb84c67..ffeae56 100644
--- a/fsl_pme/modprobe.conf
+++ b/fsl_pme/modprobe.conf
@@ -1,6 +1,6 @@
 # ************************< BEGIN COPYRIGHT >************************
 #
-# Copyright (C) 2008 Freescale Semiconductor, Inc. All rights reserved.
+# Copyright (C) 2009 Freescale Semiconductor, Inc. All rights reserved.
 #
 #   NOTICE: The information contained in this file is proprietary 
 #   to Freescale Semiconductor and is being made available to 
@@ -61,3 +61,9 @@ options pme_base freelistA=0x000000c0 freelistB=0x00000000
 #
 options ipt_l7pm use_pmchan=0
 
+# TLU settings
+options fsl_tlu log_level=0x03 tlu0_addr=0xFFE2F000 tlu0_bank0_addr=0 tlu0_bank0_size=0x00400000 tlu0_bank0_parity=0 tlu0_bank0_type=1 tlu0_bank1_addr=0 tlu0_bank1_size=0 tlu0_bank1_parity=0 tlu0_bank1_type=0 tlu0_bank2_addr=0 tlu0_bank2_size=0 tlu0_bank2_parity=0 tlu0_bank2_type=0 tlu0_bank3_addr=0 tlu0_bank3_size=0 tlu0_bank3_parity=0 tlu0_bank3_type=0 tlu1_addr=0xFFE15000 tlu1_bank0_addr=0 tlu1_bank0_size=0x00400000 tlu1_bank0_parity=0 tlu1_bank0_type=1 tlu1_bank1_addr=0 tlu1_bank1_size=0 tlu1_bank1_parity=0 tlu1_bank1_type=0 tlu1_bank2_addr=0 tlu1_bank2_size=0 tlu1_bank2_parity=0 tlu1_bank2_type=0 tlu1_bank3_addr=0 tlu1_bank3_size=0 tlu1_bank3_parity=0 tlu1_bank3_type=0
+
+options fsl_tlu_test test_case=0 debug=0
+
+
diff --git a/fsl_pme/pm_scan_demo.c b/fsl_pme/pm_scan_demo.c
index 1adfcce..dac0780 100644
--- a/fsl_pme/pm_scan_demo.c
+++ b/fsl_pme/pm_scan_demo.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/pmci.c b/fsl_pme/pmci.c
index 853a860..46c15f6 100644
--- a/fsl_pme/pmci.c
+++ b/fsl_pme/pmci.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/pme_dev_init b/fsl_pme/pme_dev_init
index 184ab4d..a018cc5 100755
--- a/fsl_pme/pme_dev_init
+++ b/fsl_pme/pme_dev_init
@@ -1,7 +1,7 @@
 #!/bin/sh
 # ************************< BEGIN COPYRIGHT >************************
 # 
-#  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+#  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
 # 
 #   NOTICE: The information contained in this file is proprietary 
 #   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/pmla_common.c b/fsl_pme/pmla_common.c
index 7044aca..3533baf 100644
--- a/fsl_pme/pmla_common.c
+++ b/fsl_pme/pmla_common.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/pmla_local.c b/fsl_pme/pmla_local.c
index 88d7f6a..bd01874 100644
--- a/fsl_pme/pmla_local.c
+++ b/fsl_pme/pmla_local.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/pmm.c b/fsl_pme/pmm.c
index 75244da..91cb627 100644
--- a/fsl_pme/pmm.c
+++ b/fsl_pme/pmm.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/pmsm.c b/fsl_pme/pmsm.c
index 1de06df..2b9dfbc 100644
--- a/fsl_pme/pmsm.c
+++ b/fsl_pme/pmsm.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/util.c b/fsl_pme/util.c
index 0517107..dc8a759 100644
--- a/fsl_pme/util.c
+++ b/fsl_pme/util.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/util.h b/fsl_pme/util.h
index db80e68..fea8bc4 100644
--- a/fsl_pme/util.h
+++ b/fsl_pme/util.h
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/util_kill.c b/fsl_pme/util_kill.c
index 2cc18c6..c67928e 100644
--- a/fsl_pme/util_kill.c
+++ b/fsl_pme/util_kill.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/util_load.c b/fsl_pme/util_load.c
index da3d42a..03d9588 100644
--- a/fsl_pme/util_load.c
+++ b/fsl_pme/util_load.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/util_monitor.c b/fsl_pme/util_monitor.c
index 9e97f4e..e7b9da0 100644
--- a/fsl_pme/util_monitor.c
+++ b/fsl_pme/util_monitor.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/util_reg.c b/fsl_pme/util_reg.c
index 0c45404..273f6fc 100644
--- a/fsl_pme/util_reg.c
+++ b/fsl_pme/util_reg.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/util_reset.c b/fsl_pme/util_reset.c
index db3712c..3af19e7 100644
--- a/fsl_pme/util_reset.c
+++ b/fsl_pme/util_reset.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/util_unload.c b/fsl_pme/util_unload.c
index 89acef7..7ae7df1 100644
--- a/fsl_pme/util_unload.c
+++ b/fsl_pme/util_unload.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
diff --git a/fsl_pme/util_writeback.c b/fsl_pme/util_writeback.c
index c07e436..74a48b9 100644
--- a/fsl_pme/util_writeback.c
+++ b/fsl_pme/util_writeback.c
@@ -1,6 +1,6 @@
 /************************< BEGIN COPYRIGHT >************************
  * 
- *  Copyright (c) 2005-2008 Freescale Semiconductor All Rights Reserved. 
+ *  Copyright (c) 2005-2009 Freescale Semiconductor All Rights Reserved. 
  * 
  *   NOTICE: The information contained in this file is proprietary 
  *   to Freescale Semiconductor and is being made available to 
-- 
1.6.0

